README.rst

```diff
@@ -14,14 +14,14 @@ TSSW Documentation Guide
 TSTN-005
 ========
 
-This document guides a member of the TSSW Team to the correct location for their documentation. 
+This document guides a member of the TSSW Team to the correct location for their documentation.
```

Comment:
I'm not thrilled with this summary. This technote seems to explain how to create a new technote.

It also has s section on configuring a regular code repo for served docs, and I don't understand if there is any connection between the two. If there isn't any connection then the summary should probably be expanded to include this thing, as well.

Reply:
I think this might be one technote to go on the delete list as the information contained here is kept elsewhere.